src/rdiff_backup/Globals.py

```diff
@@ -247,6 +247,10 @@
 # object.  Access is provided to increment error counts.
 ITRB = None
 
+# If this is set, it indicates that the remote connection should only
+# deal with paths inside of restrict_path.
+restrict_path = None  # compat200
```

Comment:
If that was missing. Most likely there is a missing test ?

Reply:
Yes and no: we have absolutely no test in our pipeline regarding cross-version testing, someone would need to think on how to do this, with a proper framework for such tests. I'm always asking people to focus their beta/RC tests on cross-version testings, apparently nobody did care so far, so here we are.